### PR TITLE
Fixes #6653: Allow windows agents

### PR DESCRIPTION
**Pull request by peckpeck**
http://www.rudder-project.org/redmine/issues/6653

### CODE REVIEW

**Comment by peckpeck** [comment]
PR updated

File: rtf [hunk 1]
```diff
@@ -137,35 +137,40 @@ class Vagrant(Host):
   def reset_platform(name, hosts):
     """ Update or replace the Vagrantfile configuration for the given platform """
     lines = []
+    # parse Vagrantfile into the lines array (and update it)
     with open("Vagrantfile", "r+") as fd:
       updated = False
```
Comment (VinceMacBuche) [inline rtf]: Just a note/question here, could we replace "updated" variable  by a "break" at the end of the while ?  or check updated in the while function?

File: vagrant.rb [hunk 2]
```diff
@@ -50,12 +57,22 @@ def configure(config, os, pf_name, pf_id, host_name, host_id, setup, version, se
...
-  command = "
-/vagrant/scripts/cleanbox #{net} \"#{host_list}\"
-export ALLOWEDNETWORK=#{net}.0/24
-/vagrant/scripts/rudder-setup setup-#{setup} \"#{version}\" \"#{server}\"
-#{if setup == "server" then "/vagrant/scripts/create-token" end}
-"
+  if os == $windows7 or os == $windows2008 then
+    command = "c:/vagrant/scripts/cleanbox.cmd #{net} \"#{host_list}\"\n"
+    if setup != "empty" then
+      command += "c:/vagrant/scripts/Rudder-agent-x64.exe /S\n"
```
Comment (VinceMacBuche) [inline vagrant.rb]: Does script "Rudder-agent-x64.exe" should be provided ?

File: rtf (hunk 1 above)
Comment (peckpeck) [inline rtf]: updated is truly a flag and not a break indicator.
We continue processing input when updated is true, to retrieve remaining lines.

File: vagrant.rb (hunk 2 above)
Comment (peckpeck) [inline vagrant.rb]: It should be provided by the user since it's not an opensource file.

Comment (VinceMacBuche) [inline vagrant.rb]: it was my guess :)

**Comment by VinceMacBuche** [comment]
Can you rebase your PR ? (not mergeable)

**Comment by peckpeck** [comment]
PR updated

**Pull request merged**